### ISSUE TITLE
Can't make it work for RGB images

### ISSUE THREAD

**Issue by LucaMarconato** [JaneliaSciComp/xarray-multiscale#13]
Does the library support `np.array` of shape `(h, w, 3)`?
The multiscale representation would operate on the first two dimensions, leaving the 3rd intact.

I have tried the code
```
        from xarray_multiscale import multiscale
        from xarray_multiscale.reducers import windowed_mean

        p = multiscale(array=data, reduction=windowed_mean, scale_factors=(2, 2, 1))
```
but it returns me an object of length 1, probably because there is no downscaling operation to perform on the last dimesion.

**Comment by d-v-b** [comment]
Thanks for the bug report! I found the issue and pushed a fix to pypi -- if you get version 0.3.1, the problem should be gone.

**Comment by LucaMarconato** [comment]
Thanks, super quick!
It works yes. 

Tested with this code

```
from xarray_multiscale import multiscale
from xarray_multiscale.reducers import windowed_mean
import numpy as np
data = np.ones((1000, 1000, 3))
p = multiscale(array=data, reduction=windowed_mean, scale_factors=(2, 2, 1))
assert len(p) > 1
```
